packages/ipfs-core/.aegir.js

```diff
@@ -15,7 +15,7 @@ let sigServerB
 let ipfsdServer
 
 module.exports = {
-  bundlesize: { maxSize: '518kB' },
+  bundlesize: { maxSize: '520kB' },
```

Comment:
Need to increase due to the addition of the [dns-over-http-resolver](https://github.com/vasco-santos/dns-over-http-resolver) module in multiaddr, which also comes with [receptacle](https://github.com/DylanPiercey/receptacle).